core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceResponseConverterTest.java

```diff
@@ -117,6 +119,11 @@ public HttpData httpData() {
                 public JsonNode jsonNode() throws IOException {
                     return JSONNODE;
                 }
+
+                @Get("/httpFile")
```

Comment:
`/http-file` for consistency? (Ditto for the path below)

Reply:
It would look better if we change all other paths into kebab-case too. Any rule for this?

Reply:
No rules, but I agree that changing to kebab-case sounds good to me.